src/guillo-gmi/actions/copy_items.js

```diff
@@ -36,10 +36,19 @@ export function CopyItems(props) {
     Ctx.flash(`Items copied!`, 'success')
   }
 
+  function getDefaultPath() {
+    const container = getContainerFromPath(Ctx.path)
+    const pathSplit = Ctx.path.split(container)
+    if (pathSplit.length > 1) {
+      return pathSplit[1]
+    }
+    return ''
+  }
+
   return (
     <PathTree
       title="Copy to..."
-      defaultPath={`/${Ctx.path.split(getContainerFromPath(Ctx.path))[1]}`}
+      defaultPath={`/${getDefaultPath()}`}
```

Comment:
@rboixaderg I think you are looking for a complicated solution for something simple (or maybe it's just me that I'm not understanding the reason if so I would like to know the use case).

 According to the documentation, `Ctx.path` exposes the current context path **without service url**.

https://github.com/guillotinaweb/guillotina_react/blob/master/docs/api.md#traversal-context

In the Guillotina component, there is already the `url` prop to add the container:


```jsx
import { Guillotina } from '@guillotinaweb/react-gmi'
// ...
return (
  <Guillotina
    config={config}
    url="/db/guillotina/" // The container is defined here
    registry={registry}
  />
)
```
This removes the need to control the `container` at all times.

So I don't see why not use the `Ctx.path` directly:

```suggestion
      defaultPath={Ctx.path}
```

Likewise, I think that perhaps all the places where `getContainerFromPath` and  `containerPath` are being used should be reviewed.

Reply:
If we set Ctx.path as defaultPath I think that is is incorrect because Ctx.path contains database and container in path, if you try duplicate an object and you put /database/container before path of object guillotina throws an error. 

For this reason I delete database and container from the path, to set a correct value in defaultValue.

Reply:
I think that we should be review all places that we use `getContainerFromPath` and `containerPath`

Reply:
mmm... The idea that was originally put forward was (@jordic confirms that this is indeed the case):

- Define the `url` with the `db` and `container` in the Guillotina component, ex: `/db/container`. https://github.com/guillotinaweb/guillotina_react/blob/master/src/guillo-gmi/components/guillotina.js#L14
- Use the `path` parameter, this way the `Ctx.path` does not have the `/db/container`. https://github.com/guillotinaweb/guillotina_react/blob/master/src/guillo-gmi/components/guillotina.js#L24

By defining the `url` on the Guillotina component, then the `RestClient` does all the requests to the correct `/db/container`:

https://github.com/guillotinaweb/guillotina_react/blob/f6da2402d9de9b92617b33a1ed23a960cfaf7ed2/src/guillo-gmi/lib/rest.js#L4

I guess you are using it in another way... I guess the default case when you don't have any of this defined... I guess we should think about whether to keep the 2 forms or force it to be done only one way.

Reply:
Yeah you can't have the `/db/container` in Ctx.path if you don't have a `path` param, but if you go to container from breadcrubs for example then you have the `path` param in url, no? 

If you don't have `path` param in url, guilltina component go to root object, in this case, database, we need to set `path` param to go container, no?

Then, the url in Guillotina component only it uses in ( I dont' see another place that we used it):

https://github.com/guillotinaweb/guillotina_react/blob/master/src/guillo-gmi/components/item.js#L30

And the url of RestClient must be absolute route, not relative, to do the requests to guillotina, no?